### PR TITLE
Updated the psr-3 spec to match the actual implementation

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: accepted/PSR-3-logger-interface.md [hunk 1]
```diff
@@ -165,7 +165,8 @@ interface LoggerInterface
      * System is unusable.
      *
      * @param string $message
-     * @param array $context
+     * @param array  $context
```
Comment (samdark) [inline accepted/PSR-3-logger-interface.md]: Double spaces?

File: accepted/PSR-3-logger-interface.md [hunk 2]
```diff
@@ -177,7 +178,8 @@ interface LoggerInterface
      * trigger the SMS alerts and wake you up.
      *
      * @param string $message
-     * @param array $context
+     * @param array  $context
```
Comment (samdark) [inline accepted/PSR-3-logger-interface.md]: Double spaces?

File: accepted/PSR-3-logger-interface.md [hunk 3]
```diff
@@ -188,7 +190,8 @@ interface LoggerInterface
      * Example: Application component unavailable, unexpected exception.
      *
      * @param string $message
-     * @param array $context
+     * @param array  $context
```
Comment (samdark) [inline accepted/PSR-3-logger-interface.md]: Double spaces?

File: accepted/PSR-3-logger-interface.md [hunk 4]
```diff
@@ -198,7 +201,8 @@ interface LoggerInterface
      * be logged and monitored.
      *
      * @param string $message
-     * @param array $context
+     * @param array  $context
```
Comment (samdark) [inline accepted/PSR-3-logger-interface.md]: Double spaces?

File: accepted/PSR-3-logger-interface.md [hunk 5]
```diff
@@ -210,7 +214,8 @@ interface LoggerInterface
      * that are not necessarily wrong.
      *
      * @param string $message
-     * @param array $context
+     * @param array  $context
```
Comment (samdark) [inline accepted/PSR-3-logger-interface.md]: Double spaces?

**Comment by samdark** [comment]
I think implementation should be updated fist removing double spaces in phpdoc.

**Comment by GrahamCampbell** [comment]
No. The double spaces are correct.

**Comment by samdark** [comment]
Why aligning in phpdoc is correct while array alignment isn't? As far as I remember it's not defined in style guide PSRs.

File: accepted/PSR-3-logger-interface.md (hunk 1 above)
Comment (GrahamCampbell) [inline accepted/PSR-3-logger-interface.md]: yes, that's correct

Comment (GrahamCampbell) [inline accepted/PSR-3-logger-interface.md]: yes, that's correct

File: accepted/PSR-3-logger-interface.md (hunk 2 above)
Comment (GrahamCampbell) [inline accepted/PSR-3-logger-interface.md]: yes, that's correct

File: accepted/PSR-3-logger-interface.md (hunk 3 above)
Comment (GrahamCampbell) [inline accepted/PSR-3-logger-interface.md]: yes, that's correct

File: accepted/PSR-3-logger-interface.md (hunk 4 above)
Comment (GrahamCampbell) [inline accepted/PSR-3-logger-interface.md]: yes, that's correct

File: accepted/PSR-3-logger-interface.md (hunk 5 above)
Comment (GrahamCampbell) [inline accepted/PSR-3-logger-interface.md]: yes, that's correct

**Comment by GrahamCampbell** [comment]
> Why aligning in phpdoc is correct while array alignment isn't? As far as I remember it's not defined in style guide PSRs.

PSR-2 doesn't say anything about array alignment, or docblocks, though PSR-5 shows all it's docblocks being formatted like this.

**Comment by GrahamCampbell** [comment]
Ping @Seldaek. So is this getting merged then, or what?

**Comment by Seldaek** [comment]
@GrahamCampbell I don't have write access here, and my understanding is that merging changes to the spec no matter how minute tends to cause noise, so ask the ml if you care :)

**Comment by GrahamCampbell** [comment]
@Seldaek Sure thing.

**Comment by GrahamCampbell** [comment]
@Seldaek How can I ask the mailing list btw? https://i.starbs.net/BN24

**Comment by Seldaek** [comment]
I guess you just need to request membership to the list first?

**Pull request closed without merging**